### ISSUE TITLE
[BUG]: Prowlarr installation

### ISSUE THREAD

**Issue by barto95100** [azukaar/Cosmos-Server#147]
### What happened?

When I test to install I received the error message:

![CleanShot 2023-11-27 at 23 07 01@2x](https://github.com/azukaar/Cosmos-Server/assets/17863254/34e53c16-8ef1-4ba7-9b79-6eef8ecc6c94)

### What should have happened?

the other app radarr,sonarr no problem to install

### How to reproduce the bug?

1. install prowlarr with default config

### Relevant log output

```shell
Creating Service...                              
Starting creation of new service...
Creating volume Prowlarr-config...
Volume Prowlarr-config created
Pulling image lscr.io/linuxserver/prowlarr:latest
latest Pulling from linuxserver/prowlarr 
{"status":"Digest: sha256:5b458ad23f7d97e5fd9bdbcc733c39c5dcab9d435d8f976d883ed49b41b45f63"}
{"status":"Status: Image is up to date for lscr.io/linuxserver/prowlarr:latest"}
Image lscr.io/linuxserver/prowlarr:latest pulled
Checking service Prowlarr...
Forcing secure Prowlarr...
Created secure network cosmos-Prowlarr-AvG
Creating container Prowlarr...
[ERROR] Rolling back changes because of -- Link network creation error: Error response from daemon: No such container: nullRolled back container Prowlarr
Rolled back network cosmos-Prowlarr-AvG
Rolled back volume Prowlarr-config
[OPERATION FAILED]. CHANGES HAVE BEEN ROLLEDBACK.
```

### System details

- OS: ubuntu 22.04 LTS
- Browser ArcBrowser
- Version cosmos: 0.13.0

**Comment by barto95100** [comment]
Ok no problem I install prowlarr manually with custom servAPP

When will version 0.14 be available?

**Comment by azukaar** [comment]
prob endof Jab

**Comment by azukaar** [comment]
Fixed in 0.14
